spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/DirectMessageListenerContainer.java

```diff
@@ -212,18 +206,6 @@ public void addQueueNames(String... queueNames) {
 		super.addQueueNames(queueNames);
 	}
 
-	@Override
-	public void addQueues(Queue... queues) {
-		try {
-			addQueues(Arrays.stream(queues)
```

Comment:
??? This is delegation to the `private void addQueues(Stream<String> queueNameStream)`.
What is the point to get rid of this logic?
Thanks